end-to-end-tests/src/main/java/com/amazonaws/blox/integ/PersonalDevelopmentStack.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.integ;
+
+import com.amazonaws.blox.dataservicemodel.v1.client.DataService;
+import com.amazonaws.blox.integ.CloudFormationStacks.CfnStack;
+import java.util.Collections;
+import java.util.List;
+import lombok.Getter;
+import software.amazon.awssdk.services.cloudformation.CloudFormationClient;
+import software.amazon.awssdk.services.dynamodb.DynamoDBClient;
+import software.amazon.awssdk.services.ecs.ECSClient;
+import software.amazon.awssdk.services.ecs.model.DescribeTasksRequest;
+import software.amazon.awssdk.services.ecs.model.DesiredStatus;
+import software.amazon.awssdk.services.ecs.model.ListTasksRequest;
+import software.amazon.awssdk.services.ecs.model.StopTaskRequest;
+import software.amazon.awssdk.services.ecs.model.Task;
+import software.amazon.awssdk.services.lambda.LambdaAsyncClient;
+
+/** Facade over everything in an end-to-end Blox stack that's needed in tests. */
+public class PersonalDevelopmentStack implements TestStack {
+  private CloudFormationClient cloudformation = CloudFormationClient.create();
+  private DynamoDBClient ddb = DynamoDBClient.create();
+  private final DynamoDBSupport ddbSupport = new DynamoDBSupport(ddb);
+  private ECSClient ecs = ECSClient.create();
+  private LambdaAsyncClient lambda = LambdaAsyncClient.create();
+  private CloudFormationStacks stacks = new CloudFormationStacks(cloudformation);
+  private final DataServiceSupport dataServiceSupport = new DataServiceSupport(lambda, stacks);
+  private final CfnStack testCluster = stacks.get("blox-test-cluster");
+  @Getter private String taskDefinition = testCluster.output("taskdef");
+  @Getter private String cluster = testCluster.output("cluster");
+
+  @Override
+  public DataService createDataService() {
+    return dataServiceSupport.createDataService();
+  }
+
+  @Override
+  public List<Task> describeBloxTasks() {
+    List<String> taskArns = listBloxTasks();
+    if (taskArns.isEmpty()) return Collections.emptyList();
+
+    return ecs.describeTasks(
+            DescribeTasksRequest.builder().cluster(cluster).tasks(taskArns).build())
+        .tasks();
+  }
+
+  private List<String> listBloxTasks() {
```

Comment:
the blox tasks shouldn't be in the dev stack right? this should just set everything up and the tests implement the verificaiton? so there's separation between just stack setup and tests.

Reply:
Yeah, this really should be pulled out into an ECS wrapper class instead of being stuck in here. Will do.